index.html

```diff
@@ -74,8 +74,12 @@ <h1 style="text-align: center">Godot proposals viewer</h1>
           <option value="oldest" title="Sort proposals by oldest first">Oldest</option>
         </select>
         <a :href="`https://github.com/godotengine/godot-proposals/issues${(labelFilter !== '' ? `/labels/${labelFilter}` : '')}`" target="_blank" rel="noreferrer noopener" style="margin-left: 1rem" >
-          <span x-text="`${proposalsFiltered().length} open proposals`"></span>
+          <span x-text="`${proposalsFiltered(true).length} open proposals`"></span>
```

Comment:
since open proposals' link only includes filtered label, it excludes the search query through the excludeSearch parameter